src/components/elements/wysiwyg.tsx

```diff
@@ -52,6 +60,20 @@ const options: HTMLReactParserOptions = {
           )
 
         case "div":
+          if (nodeProps.className && nodeProps.className.includes("left-bar")) {
+            nodeProps.className = twMerge(
+              nodeProps.className,
+              "max-w-[725px] rs-pb-5 rs-pt-7 rs-pl-1 border-3 border-archway-dark border-opacity-50"
+            )
+            return <NodeName {...nodeProps}>{domToReact(children, options)}</NodeName>
+          }
+
+          delete nodeProps.role
+          if (nodeProps.className && !!nodeProps.className.indexOf("media-entity-wrapper")) {
+            return cleanMediaMarkup(domNode)
+          }
+          return <NodeName {...nodeProps}>{domToReact(children, options)}</NodeName>
```

Comment:
You don't need these because if you don't have a return, it'll go down into the `article` logic.